### PR TITLE
4.0.0: Various fixes

### PR DESCRIPTION
**Pull request by Kenny-Hui**
- Fix bridge creator speed by blacklisting the appropriate position (#747) (Courtsey of MTR-NTE: https://github.com/zbx1425/mtr-nte/blob/eaeded917d99c3939db25a9587630f0b390ece84/common/src/main/java/cn/zbx1425/mtrsteamloco/mixin/RailActionsMixin.java)
- Add Flight Dashboard Recipe (Feather) (#736)
- Improve Dashboard map performance by not rendering black which is same as the background (Same as #768)
- Don't show scientific notation for very large coordinates in Widget Map
- Fix floating point precision error preventing player from clicking on Widget Map on far distances
- Generate Fabric Run Configuration for IDE, allowing Debugging and Live Reload
- Rail Action description percentage are now rounded to 1 decimal place, the block used are also translated.
- ~Fix Station Colored Glass and Glass Slab see-through blocks in 4.0.0~ (Already fixed)

### CODE REVIEW

**Comment by jonafanho** [comment]
Thanks, don't know why I didn't see this earlier lol
I will review and merge soon

File: fabric/build.gradle [hunk 1]
```diff
@@ -10,6 +10,9 @@ final BuildTools buildTools = new BuildTools(minecraftVersion, "fabric", project
 
 loom {
 	accessWidenerPath = file("src/main/resources/mtr.accesswidener")
+	runConfigs.configureEach {
+		ideConfigGenerated = true
+	}
```
Comment (jonafanho) [inline fabric/build.gradle:15]: I couldn't get the generated Fabric client run config to work in Intellij, did you have any success? I am removing this for now.

Comment (Kenny-Hui) [reply fabric/build.gradle:15]: It works here (At least in the making of this PR), although IIRC I need to restart IntelliJ for it to not crash on launch

If that doesn't work it's fine if we remove this for now

Comment (jonafanho) [reply fabric/build.gradle:15]: Deleting the `.idea` folder refreshed the cache and did the trick, thanks!

**Pull request merged**